packages/teams-js/src/public/location.ts

```diff
@@ -1,14 +1,12 @@
-import { sendAndHandleSdkError as sendAndHandleError } from '../internal/communication';
+import { sendAndHandleSdkError as sendAndHandleError, sendMessageToParent } from '../internal/communication';
 import { locationAPIsRequiredVersion } from '../internal/constants';
 import { ensureInitialized, isCurrentSDKVersionAtLeast } from '../internal/internalAPIs';
-import {
-  callCallbackWithErrorOrBooleanFromPromiseAndReturnPromise,
-  callCallbackWithErrorOrResultFromPromiseAndReturnPromise,
-} from '../internal/utils';
-import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
-import { ErrorCode, SdkError } from './interfaces';
+import { FrameContexts } from './constants';
+import { DevicePermission, ErrorCode, SdkError } from './interfaces';
 import { runtime } from './runtime';
-
+/**
+ * @alpha
```

Comment:
I seem to remember erin saying that @alpha doesn't actually *mean* anything in our documentation. Given that location is already a shipped capabiility that we are restructuring, we should probably just leave @alpha/@beta off of this

Reply:
(Expanding this comment to say what this capability is would be great as well)

Reply:
Will do thank you!